modules/ROOT/pages/faq.adoc

```diff
@@ -310,3 +310,27 @@ xref:storage.adoc[Configuring Storage] page for examples.
 
 See the xref:faq.adoc#_why_is_the_dnsmasq_service_systemd_unit_masked[Why is the `dnsmasq.service` systemd unit masked]
 entry for an example config to unmask this unit.
+
+== How do I add the missing wifi-firmwares ?
+
+Some wifi-firmwares will be split into sub-packages in fedora 40.
+Fedora Cores OS will keep them in until fedora 41, but display a
+warning message in the console if NetworkManager-wifi is layered without
+any other wifi firmwares.
+
+If NetworkManager-wifi is a requested layered package, a warning is
+displayed to encourage users to also add a wifi-firmware.
+
+.Install thre required firmware for your hardware:
+[source, text]
+----
+$ sudo rpm-ostree install iwlwifi-dvm-firmware
+----
+
+Then reboot the system. The warning should go away.
+
+You can always disable the warning with:
+[source, text]
+----
+sudo systemctl disable coreos-check-wireless-firmwares.service
+----
```

Comment:
I would just say to reboot the system and not worry about saying the warning will go away. 
```suggestion
Then reboot the system.
```

Reply:
actually no.. let's tell them to always disable the check so it won't run on every boot (i.e. they have taken necessary action now).

so tell them to run the `systemctl disable` and then reboot the system